app/views/datagrid/_table.html.erb

```diff
@@ -5,18 +5,18 @@ Local variables:
 * options - passed options Hash
 %>
 <% if grid.html_columns(*options[:columns]).any? %>
-  <%= content_tag :table, options[:html] do %>
+  <%= content_tag :table, class: 'datagrid-table', **options.fetch(:html, {}) do %>
```

Comment:
Consider changing views to use shorter `tag` helper added in Rails 5.1.
https://www.bigbinary.com/blog/new-syntax-for-tag-helpers-in-rails-5-1
```suggestion
  <%= tag.table, class: 'datagrid-table', **options.fetch(:html, {}) do %>
```

Reply:
On cases like this feel myself a retro rails developer.